CMakeLists.txt

```diff
@@ -480,24 +480,24 @@ set(SOURCES
         db/c.cc
         db/column_family.cc
         db/compacted_db_impl.cc
-        db/compaction.cc
-        db/compaction_iterator.cc
-        db/compaction_job.cc
-        db/compaction_picker.cc
-        db/compaction_picker_fifo.cc
-        db/compaction_picker_level.cc
-        db/compaction_picker_universal.cc
+        db/compaction/compaction.cc
+        db/compaction/compaction_iterator.cc
+        db/compaction/compaction_picker.cc        
+        db/compaction/compaction_job.cc
```

Comment:
I know it doesn't go with the name compaction but I feel it natural if flush_job.* are kept together with compaction_job.*.

Reply:
discussed offline will leave it separate.